### PR TITLE
Add support to show popovers on page load

### PR DESCRIPTION
**Pull request by dnoler**

### CODE REVIEW

File: lib/ui/popover/popover.js [hunk 1]
```diff
@@ -44,12 +44,26 @@
...
     };
+
+    if($scope.showOnLoad) {
+      this.show();
+      if($scope.delay && $scope.delay.hide) {
+        $timeout(this.hide, $scope.delay.hide);
+        return;
+      }
+      // If no delay is found or cannot be parsed, set a default timeout so that the popover doesn't stick around forever
+      $timeout(this.hide, 10000);
```
Comment (robmcguinness) [inline lib/ui/popover/popover.js]: `$timeout(this.hide, 10000, false);` - If set to false skips model dirty checking.  This is typically a performance boost if your function `this.hide` doesn't depend on data that may have changed.  Let me know your thoughts

Comment (robmcguinness) [inline lib/ui/popover/popover.js]: I would probably move out the default delay `10000` into some kind of configuration that can be modified by other users of the SDK.  

Here is an example that uses a provider for configuring all dropdowns globally across the app [avDropdownConfig ](https://github.com/Availity/availity-angular/blob/31e184a27ed5eadd739c935900c3eec869716d07/lib/ui/dropdown/dropdown.js#L7)

Comment (dnoler) [inline lib/ui/popover/popover.js]: We definitely don't depend on any data changes for the hide function, I'll add the false flag. I also agree with your second comment, I'll add a config provider.

File: lib/ui/popover/popover.js [hunk 2]
```diff
@@ -44,12 +59,26 @@
...
     };
+
+    if($scope.showOnLoad) {
+      this.show();
+      if($scope.delay && $scope.delay.hide) {
+        $timeout(this.hide, $scope.delay.hide, false);
+        return;
+      }
+      // If no delay is found or cannot be parsed, set a default timeout so that the popover doesn't stick around forever
+      $timeout(this.hide, this.options.showOnLoadHideDelay);
```
Comment (robmcguinness) [inline lib/ui/popover/popover.js]: `$timeout(this.hide, this.options.showOnLoadHideDelay, false);` as well?

**Pull request merged**